### PR TITLE
P1 features

### PR DESCRIPTION
**Pull request by SukanyaSen530**
Functionalities -
1.  Task Priority
   - User can select task priority in the task form and it show up in task card - low -- green, yellow -- medium, red -- high
2.  Sessions -
   - Once the user starts the timer, work and Short Break mode will toggle automatically until one session completes - (2 sessions)
   - It will automatically switch to Long Break MODE once session is completed.
   - Audio will be played once a mode is witched
3. Tags-
  - User can add/remove multiple tags in the task detail page
4. Completed and uncompleted task are filtered in the tasks page.

### CODE REVIEW

File: src/services/tasks.js [hunk 1]
```diff
@@ -128,3 +128,30 @@ export const toggleTaskCompletion = async (id, dispatch, setCheckLoading) => {
...
+  try {
+    const { data, status } = await axios.put(
+      `${tasksURL}/tags/${id}`,
+      { tags },
+      config
+    );
+
+    if (status === 200) {
+      setTagsLoading(false);
+      console.log("data", data);
```
Comment (parthgabani777) [inline src/services/tasks.js:145]: You can remove the console if not necessary.

**Review by parthgabani777 — APPROVED** [review]

File: src/components/InputField/InputField.jsx [hunk 2]
```diff
@@ -3,32 +3,48 @@ import "./input-field.scss";
 import { useState } from "react";
 import { AiFillEye, AiFillEyeInvisible } from "react-icons/ai";
 
-function InputField({ labelName, type, value, name, errorMessage, ...other }) {
+function InputField({
```
Comment (animeshgarai09) [inline src/components/InputField/InputField.jsx:6]: Use arrow functions for your components

**Review by animeshgarai09 — APPROVED** [review]

File: src/services/tasks.js (hunk 1 above)
Comment (SukanyaSen530) [reply src/services/tasks.js:145]: Thanks for pointing out, will remove it in next PR

File: src/components/InputField/InputField.jsx (hunk 2 above)
Comment (SukanyaSen530) [reply src/components/InputField/InputField.jsx:6]: Thanks for pointing out, will update it in next PR

**Pull request merged**